config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'root/index'
```

Comment:
## [OPTIONAL]

- [ ] For best pratics I thing it always better to have a good structure of the API endpoints you can use namespace to have a good tructure and help you control version of your API in ROR, check this example
```suggestion
  namespace :api do
    namespace :v1 do
      resources :users
      resources :posts
    # ... other resources
  end

  namespace :v2 do
      resources :users
      resources :posts
    # ... other resources
  end
end
```